### PR TITLE
fix: replace static target and member count with dynamic counts

### PR DESCRIPTION
**Pull request by dabick14**
## Description

This PR replaces the static member count and targets.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?
This has been tested on my localhost

## Screenshots/Videos (if appropriate):
image

### CODE REVIEW

File: api/src/schema/arrivals.graphql [hunk 1]
```diff
@@ -1,19 +1,43 @@
 extend type GatheringService {
   target: Int!
+    @cypher(
+      statement: """
+      MATCH (this)-[:HAS*4]->(bacenta:Bacenta)
+      RETURN SUM(DISTINCT bacenta.target)
```
Comment (jaedag) [inline api/src/schema/arrivals.graphql:6]: Why the use of the word distinct here?

**Review by jaedag — APPROVED** [review]

**Review by jaedag — CHANGES_REQUESTED** [review]
Why the use of the distinct when calculating target

File: api/src/schema/arrivals.graphql (hunk 1 above)
Comment (dabick14) [reply api/src/schema/arrivals.graphql]: Removing it now

**Comment by dabick14** [comment]
> Why the use of the distinct when calculating target 

Removed now

**Comment by jaedag** [comment]
That's fine but if you can please answer the question @dabick14

**Comment by dabick14** [comment]
> That's fine but if you can please answer the question @dabick14 

I was copying and pasting some code. I should have seen that there was no use for the distinct keyword there.

**Review by jaedag — APPROVED** [review]

**Pull request merged**